src/common/iop_order.c

```diff
@@ -663,14 +663,14 @@ static void _ioppr_reset_iop_order(GList *iop_order_list)
   }
 }
 
-void dt_ioppr_resync_iop_list(dt_develop_t *dev)
+void dt_ioppr_resync_iop_list(dt_develop_t *const dev)
 {
   // make sure that the iop_order_list does not contains possibly removed modules
 
   GList *l = g_list_first(dev->iop_order_list);
   while(l)
   {
-    GList *next = g_list_next(l);
+    GList *const restrict next = g_list_next(l);
```

Comment:
This makes no difference,  because nothing is accessed through that pointer, so you're just adding noise. Also if something was accessed through it, it would be wrong, because list is later modified through another pointer (`dev->iop_order_list`).
Plus, not part of this patch, but 6 lines below there's a memory leak - g_list_remove_link doesn't free the removed element of the list.